src/components/new-feature.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { Link } from 'gatsby'
+
+// import "./footer.scss"
+import ArrowRightAlt from '@material-ui/icons/ArrowRightAlt';
+
+
+const NewFeature = (props) => {
+
+    return (
+        <div className="mt-4">
+            <h3>{props.heading}</h3>
+            <div>{props.date}</div>
+            <img src={props.image} className="mw-100 pt-3"/>
```

Comment:
Problem from my code... there is a warning that we should have `alt` text for accessibility. My plan was to have this content coming from the Contentful CMS in the next phase. Could we just comment out the image for now and we can handle images then?